BookingsApi/BookingsApi.DAL/Migrations/20220405123231_AddTrackingDateTimeColumns.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddTrackingDateTimeColumns : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedDate",
+                table: "UserRole",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```

Comment:
Perhaps this should be consistent across your other PRs. ie the dates should all default to a datetime, or none of them should